bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -188,6 +188,43 @@ spec:
           containerPort: {{.OVN_NB_PORT}}
         - name: nb-db-raft-port
           containerPort: {{.OVN_NB_RAFT_PORT}}
+        livenessProbe:
```

Comment:
No liveness probes for now - they are really hard to get right and usually break things.